.changeset/smart-turkeys-bathe.md

```diff
@@ -0,0 +1,30 @@
+---
+'@backstage/plugin-catalog-backend': minor
+'@backstage/plugin-catalog-import': minor
+'example-app': patch
```

Comment:
don't think you need to bump either this or the `example-backend`

Reply:
You think that I shouldn't bump patch versions of example-app and example-backend? I've modified files in `packages/app` and `packages/backend` directories so thought that it means they need a version bump.

Reply:
Not needed - they are private packages that aren't released; they are essentially like unversioned

Reply:
I've changed it so example-app and example-backend are not updated